crates/sui-json-rpc/src/error.rs

```diff
@@ -133,10 +134,11 @@ impl From<Error> for RpcError {
             Error::QuorumDriverError(err) => {
                 match err {
                     QuorumDriverError::InvalidUserSignature(err) => {
+                        let error_weight = Weight::from(err.clone());
                         let error_object = ErrorObject::owned(
                             TRANSACTION_EXECUTION_CLIENT_ERROR_CODE,
                             format!("Invalid user signature: {err}"),
-                            None::<()>,
+                            serde_json::to_string(&error_weight).ok(),
```

Comment:
i think we should use an evolvable format in here. i'm not too familiar with the rust json stuff and so i'm not sure whether a simple struct would be okay. it would depend on the deserializer ignoring unknown fields.